h/migrations/versions/8ae9d103551f_backfill_group_access_flags.py

```diff
@@ -46,9 +46,9 @@ class WriteableBy(enum.Enum):
 def upgrade():
     op.execute(
         group_table.update().values(
-            joinable_by=JoinableBy.authority,
-            readable_by=ReadableBy.members,
-            writeable_by=WriteableBy.members,
+            joinable_by=JoinableBy.AUTHORITY,
+            readable_by=ReadableBy.MEMBERS,
+            writeable_by=WriteableBy.MEMBERS,
```

Comment:
Did we fix the SQL generation issue with this? This doesn't look like the definition of the Enum?

Reply:
No, rebase fail